src/test/resources/metadata-manager/inventory_instance_with_2_items.json

```diff
@@ -61,6 +62,7 @@
         },
         "enumeration": "Enumeration 1_2_1",
         "materialType": "book",
+        "copyNumber": "copy number",
```

Comment:
This is on the inventory guys not you... seems silly to all it copyNumber when it's not a number

Reply:
It was just a string for testing purposes